pytorch_lightning/plugins/training_type/sharded_spawn.py

```diff
@@ -115,12 +114,12 @@ def pre_backward(self, closure_loss: torch.Tensor) -> None:
     def post_training_step(self):
         pass
 
-    def new_process(self, trainer: "pl.Trainer", mp_queue: SimpleQueue) -> None:
+    def new_process(self, trainer: "pl.Trainer") -> Optional[Tuple[Optional[str], Optional[str], Any, _FakeQueue]]:
```

Comment:
Same comment

Reply:
true, I forgot to update the subclasses. Fortunately new_process will become obsolete in 2/n if everything goes according to plan.

Reply:
resolved in #10934